src/components/common/CodeReview.js

```diff
@@ -0,0 +1,228 @@
+  '        try:\n' +
+  "            role = await ctx.guild.create_role(name=role_name, reason='New course channel')\n" +
+  '            # get category\n' +
+  '            course_category = discord.utils.get(ctx.guild.categories, id=cfg.COURSE_CHANNEL_ID)\n' +
+  '            overwrite_rules = {role: discord.PermissionOverwrite(read_messages=True),\n' +
+  '                               ctx.guild.default_role: discord.PermissionOverwrite(read_messages=False)}\n' +
+  '            await ctx.guild.create_text_channel(name=course_name, category=course_category, overwrites=overwrite_rules)\n' +
+  '            await ctx.message.author.add_roles(role)\n' +
+  '        except (discord.Forbidden, discord.HTTPException):\n' +
+  "            await u_msg.send_error(ctx, f'Woops something went wrong, contacting <@!{cfg.DEV_ID}>')\n" +
+  '        else:\n' +
+  "            await ctx.message.add_reaction('✅')\n" +
+  '\n' +
+  "    @course.command(name='sub', aliases=['join', 'rejoindre'], help='Subscribe to a course channel')\n" +
+  '    async def sub_course(self, ctx, course_name):\n' +
+  '        # Check if role is authorized\n' +
+  '        member = ctx.message.author\n' +
+  "        new_role = discord.utils.get(member.guild.roles, name='sub2:' + course_name)\n" +
+  '        if new_role:\n' +
+  '            try:\n' +
+  '                await member.add_roles(new_role)\n' +
+  '            except (discord.Forbidden, discord.HTTPException):\n' +
+  "                await u_msg.send_error(ctx, f'Woops something went wrong, contacting <@!{cfg.DEV_ID}>')\n" +
+  '            else:\n' +
+  "                await ctx.message.add_reaction('✅')\n" +
+  '        else:\n' +
+  "            await ctx.send(f'There is no course named : {course_name}.')\n" +
+  '\n' +
+  "    '''\n" +
+  '    # not quite\n' +
+  '    @delete_course.error\n' +
+  '    async def del_course_error(error, ctx):\n' +
+  '        if isinstance(error, discord.ext.commands.MissingPermissions):\n' +
+  "            await msg_error(ctx, 'You are lacking the necessary permissions. Sorry.')\n" +
+  "    '''\n";
+
+class CodeReview extends Component {
+  static propTypes = {
+    classes: PropTypes.shape({
+      root: PropTypes.string,
+      table: PropTypes.string,
+      actions: PropTypes.string,
+      button: PropTypes.string,
+    }).isRequired,
+  };
+
+  static defaultProps = {};
+
+  static styles = (theme) => ({
+    root: {
+      padding: theme.spacing(1),
+      margin: theme.spacing(1),
+    },
+    button: {
+      marginTop: theme.spacing(3),
+    },
+    actions: {
+      justifyContent: 'flex-end',
+    },
+  });
+
+  static highlightCode(code, syntax) {
+    let highlighted;
+    if (syntax === 'python') {
+      highlighted = Prism.highlight(code, Prism.languages.python, 'python');
+    } else {
+      highlighted = Prism.highlight(
+        code,
+        Prism.languages.javascript,
+        'javascript',
+      );
+    }
```

Comment:
Instead of this you could use `highlighted = Prism.highlight(code, Prism.languages[syntax], syntax)`.